examples/spark-data-lake/infra/stacks/application_stack.py

```diff
@@ -61,21 +60,21 @@ def __init__(
                         "s3:Abort*"
                     ],
                     resources=[
-                        f"{storage.bronze_bucket.bucket_arn}/*",
                         f"{storage.silver_bucket.bucket_arn}/*",
+                        f"{storage.gold_bucket.bucket_arn}/*",
                         storage.silver_bucket.bucket_arn,
-                        storage.bronze_bucket.bucket_arn
+                        storage.gold_bucket.bucket_arn
```

Comment:
You can remove this now, the `createExecutionRole` in EmrServerlessRuntime return a role with no policy provided.